### DISCUSSION TITLE
redundant "WebGraph" in  webgraph::traits::WebGraphCodes{Reader,Writer}

### DISCUSSION THREAD

**Discussion by zacchiro** [vigna/webgraph-rs#18] [Ideas]
Question (not an issue because maybe it's by design): do we want "WebGraph" to appear twice in the fully-qualified names of the traits `webgraph::traits::WebGraphCodesReader` and `webgraph::traits::WebGraphCodesWriter`.

FWIW, those seems to be the only two traits in the `webgraph::traits` module that show this redundancy.

**Comment by vigna** [comment]
True. Also, I thought the new naming fad was actions ("CodesRead" instead of "CodesReader"). But in fact that is a BV reader—the name should be changed at some point.
